test/test_core.py

```diff
@@ -675,3 +676,39 @@ def test_core_list_platform_without_platform_txt(run_command, data_dir):
     core = cores[0]
     assert core["id"] == "some-packager:some-arch"
     assert core["name"] == "some-packager-some-arch"
+
+
+def test_core_with_wrong_custom_board_options_is_loaded(run_command, data_dir):
+    test_platform_name = "platform_with_wrong_custom_board_options"
+    platform_install_dir = Path(data_dir, "hardware", "arduino-beta-dev", test_platform_name)
+    platform_install_dir.mkdir(parents=True)
+
+    # Install platform in Sketchbook hardware dir
+    shutil.copytree(
+        Path(__file__).parent / "testdata" / test_platform_name, platform_install_dir, dirs_exist_ok=True,
+    )
+
+    assert run_command("update")
+
+    res = run_command("core list --format json")
+    assert res.ok
+
+    cores = json.loads(res.stdout)
+    mapped = {core["id"]: core for core in cores}
+    assert len(mapped) == 1
+    # Verifies platform is loaded except excluding board with wrong options
+    assert "arduino-beta-dev:platform_with_wrong_custom_board_options" in mapped
+    boards = {b["fqbn"]: b for b in mapped["arduino-beta-dev:platform_with_wrong_custom_board_options"]["boards"]}
+    assert len(boards) == 1
+    # Verify board with malformed options is not loaded
+    assert "arduino-beta-dev:platform_with_wrong_custom_board_options:nessuno" not in boards
+    # Verify other board is loaded
+    assert "arduino-beta-dev:platform_with_wrong_custom_board_options:altra" in boards
+    # Verify warning is shown to user
+    assert (
+        "Error initializing instance: "
+        + "loading platform release arduino-beta-dev:platform_with_wrong_custom_board_options@4.2.0: "
+        + "loading boards: "
+        + "skip loading of boards arduino-beta-dev:platform_with_wrong_custom_board_options:nessuno: "
```

Comment:
```suggestion
        + "skipping loading of boards arduino-beta-dev:platform_with_wrong_custom_board_options:nessuno: "
```
This accompanies my suggestion above.